inz-id-card/Issues/address/description.xml

```diff
@@ -0,0 +1,33 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo INZ Address</en>
+        <nl>Demo INZ Address</nl>
+    </Name>
+    <ShortName>
+        <en>Address</en>
+        <nl></nl>
+    </ShortName>
+    <SchemeManager>irma-demo</SchemeManager>
+    <IssuerID>inz-id-card</IssuerID>
+    <CredentialID>address</CredentialID>
+    <Description>
+        <en>Full address</en>
+        <nl></nl>
+    </Description>
+    <IssueURL>
+        <en>https://demo.irma.nz/id-card/</en>
+        <nl>https://demo.irma.nz/id-card/</nl>
```

Comment:
This does not seem to resolve to a webpage where an IRMA app user can obtain or refresh these cards. In demo this is fine by me, but it does mean that the refresh option reachable from the context menu in the top right of the cards will be broken.

Reply:
Good catch! It's fixed now.